### PR TITLE
CPOL-147 Rename project to "Policies"

### PR DESCRIPTION
**Pull request by josejulio**
image

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,21 +8,21 @@ cache:
...
+- yarn
+- yarn lint
+- yarn build
 - yarn test --forceExit --maxWorkers=4
 - yarn deploy && curl -sSL https://raw.githubusercontent.com/RedHatInsights/insights-frontend-builder-common/master/src/bootstrap.sh | bash -s
 env:
  global:
- - REPO="git@github.com:RedHatInsights/custom-policies-ui-frontend-build"
- - REPO_DIR="custom-policies-ui-frontend-build"
+ - REPO="git@github.com:RedHatInsights/policies-ui-frontend-build"
```
Comment (josejulio) [inline .travis.yml]: @ryelo We are renaming from `custom-policies` to `policies`.

Could we get our *-build branch renamed to make them consistent ?

Comment (ryelo) [reply .travis.yml]: Is this really something we need to update across the board? Changing the repo names won't affect anything on the actual UI, that's all handled in cloud services config

Comment (josejulio) [reply .travis.yml]: We want to keep it consistent on every project / system to make everything easier to find.

File: src/utils/RbacUtils.test.ts [hunk 2]
```diff
@@ -5,7 +5,7 @@ describe('src/utils/RbacUtils', () => {
     it('RawRbac detects fullAccess', () => {
         const fullAccess: Access[] = [
             {
-                permission: 'custom-policies:*:*',
+                permission: 'policies:*:*',
```
Comment (josejulio) [inline src/utils/RbacUtils.test.ts]: Updating rbac stuff too

File: .travis.yml (hunk 1 above)
Comment (ryelo) [reply .travis.yml]: Sure thing. We just need to coordinate it with the cloud services config pull. I'm not sure, but I may have to regen the ssh key, so there might be other PRs I need to make here.

**Comment by ryelo** [comment]
You also need updated:
https://github.com/RedHatInsights/custom-policies-ui-frontend/blob/404ebd4f598a7f275c9097c2ffb2b12c298799f3/profiles/local-frontend-and-api.ts#L4
https://github.com/RedHatInsights/custom-policies-ui-frontend/blob/404ebd4f598a7f275c9097c2ffb2b12c298799f3/profiles/local-frontend.ts#L2
https://github.com/RedHatInsights/custom-policies-ui-frontend/blob/2854a1cc584d2a9aaa80076d6845d7c6bde15d40/package.json#L111

**Comment by ryelo** [comment]
I will be updating the build repo on March 25 and then we can trigger a build to make sure everything works right.

**Pull request merged**